pkg/controller/kogitoruntime/deployer.go

```diff
@@ -181,35 +170,6 @@ func getProtoBufConfigMapName(serviceName string) string {
 	return fmt.Sprintf("%s-%s", serviceName, protobufConfigMapSuffix)
 }
 
-// applyProtoBufConfigurations configures the deployment to handle protobuf
-func applyProtoBufConfigurations(deployment *v1.Deployment, kogitoService v1alpha1.KogitoService) {
```

Comment:
Why is this function removed?

Reply:
We don't need the Downard API anymore since we removed the protobuf script from the image. That was used by the image to update the Protobuf Config Map.